README.md

```diff
@@ -46,7 +46,7 @@ $ build/Darwin.x86_64.Debug/game/game
 #### Ubuntu
 
 ```
-$ apt-get install cmake libsdl2-dev libsdl2-ttf-dev liblua5.2-dev libboost-dev
+$ apt-get install cmake libsdl2-dev libsdl2-ttf-dev liblua5.2-dev libboost-system-dev libboost-filesystem-dev
```

Comment:
Could you rebase to the current master? It was changed today to install `libboost1.58-all-dev` from  `libbost-dev`.

Reply:
It still seem liboost-dev at master branch.

Reply:
You're right. Thanks!